web-client/src/views/WorkQueue/SectionWorkQueueInbox.tsx

```diff
@@ -131,21 +132,11 @@ SectionWorkQueueTable.Row = React.memo(
             </td>
           )}
           <td className="consolidated-case-column">
-            {item.inConsolidatedGroup && (
-              <span
-                className="fa-layers fa-fw"
-                title={item.consolidatedIconTooltipText}
-              >
-                <Icon
-                  aria-label={item.consolidatedIconTooltipText}
-                  className="fa-icon-blue"
-                  icon="copy"
-                />
-                {item.inLeadCase && (
-                  <span className="fa-inverse lead-case-icon-text">L</span>
-                )}
-              </span>
-            )}
+            <ConsolidatedCaseIcon
```

Comment:
🔧 To me, this reads as though this icon will be rendered for each item in the table, would it be possible to extract the logic that determines whether or not to render the component outside of the component itself?

Reply:
We went back and forth on this for a little bit. We ended up putting the show/hide logic inside of the ConsolidatedCaseIcon.tsx so that it was easier for other components to use. Not sure which way we prefer.